### ISSUE TITLE
Dissection of the internal carotid artery

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7041]
**Preferred term label:**
Internal carotid artery dissection

**Synonyms**
ICA dissection

**Definition (free text, please give PubMed ID)**
A sudden tear in the intimal layer of the internal carotid artery allows blood to flow into the intimal layer of the vessel leading to the formation of a hematoma within the blood vessel wall. This is commonly known as a false lumen. As blood enters this false lumen, it causes stenosis which may lead to complete occlusion of the carotid artery. This is a dynamic process which may lead to stenosis of the vessel or dilatation of the artery. 
PMID:28613585

**Parent term (use hpo.jax.org/app)**
Abnormal internal carotid artery morphology HP:3000062

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:618734  1/17  PMID: 27895300

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
